client/src/app/submitter/project-form/layout.json

```diff
@@ -1,11 +1,22 @@
 {
   "tabs": [
+    {
+      "title": "General Info",
+      "items": [
+        "project.dataAccess",
+        "project.identifyingOrganisms",
+        "project.technology",
+        "project.releaseDate",
+        "project.accessionDate"
+      ]
+    },
     {
       "title": "Project",
       "items": [
         "project.content.project_core.project_title",
         "project.content.project_core.project_description",
         "project.content.project_core.project_short_name",
```

Comment:
The project tab should be first or at least we need to keep the title and short name on the first page.
This is so that when the user clicks "save and continue" the first time the project is made with the minimum info of title, short name, description.

Reply:
People seem to have different opinion on this. I just merged the fields in the Project Tab for now.

Reply:
I was coming at it from the point of view of avoiding the bug of: "if you don't have a short name you can't click the project in the list 😀"
And also avoiding lots of projects with empty names